src/tool/python/strings/setup.py

```diff
@@ -1,17 +1,16 @@
-# WARNING: Please don't edit this file. It was generated by Python/WinRT
-
 # TODO: remove explicit debug settings in favor of build_ext --debug setting
 
 import setuptools
 
 setuptools.setup(
     name = "%",
-    version = "1.0.alpha",
+    version = "1.0a0",
     description="Generated Python/WinRT package",
     license="MIT",
     url="http://github.com/Microsoft/xlang",
     ext_modules = [ setuptools.Extension('%', 
         sources = [%],
+        py_limited_api = True,
```

Comment:
This will likely lead to more problems than it solves, as it will let you compile use of many Python APIs that are not available in the `python3.dll` it links to. Better off leaving it out and letting the compiled pyd be Python version-specific.

Reply:
will do